tests/dsl/dtd/CMakeLists.txt

```diff
@@ -32,5 +32,10 @@ if( PARSEC_HAVE_CUDA )
       target_include_directories(dtd_bench_simple_gemm PRIVATE ${BLAS_INCLUDE_DIRS})
       target_compile_definitions(dtd_bench_simple_gemm PRIVATE HAVE_BLAS=1)
     endif(BLAS_FOUND)
+    find_package(CUDAToolkit)
+    enable_language(CUDA)
```

Comment:
Why do we need this here in the tests ? It should have been enabled at a higher level.